src/renderer/gl_engine/tvgGlCommon.h

```diff
@@ -26,12 +26,17 @@
 #include <assert.h>
 #if defined (THORVG_GL_TARGET_GLES)
     #include <GLES3/gl3.h>
+    #define TVG_REQUIRE_GL_MAJOR_VER 3
+    #define TVG_REQUIRE_GL_MINOR_VER 0
 #elif defined (THORVG_GL_TARGET_GL)
     #if defined(__APPLE__) || defined(__MACH__)
         #include <OpenGL/gl3.h>
     #else
-        #include <GL/gl3.h>
+        #define GL_GLEXT_PROTOTYPES 1
+        #include <GL/gl.h>
```

Comment:
@RuiwenTang I couldn't find `<GL/gl3.h>` please confirm this, Thanks.

Reply:
which platform do you compile to ?

Reply:
linux(ubuntu)

Reply:
I will check this on my Ubuntu environment.

BTW:
Maybe we should consider dynamically loading GL functions, which would avoid most compilation and linking issues